lib/lackeys/rails_base.rb

```diff
@@ -37,5 +37,21 @@ def method_missing(method_name, *args, &block)
         super
       end
     end
+
+    #######################
+    # Development methods #
+    #######################
+    def who_has?(method_name)
+      return nil unless respond_to?(method_name)
+      res = registry.method?(method_name, true)
```

Comment:
We might want to keep `#method?` to be boolean (which is what you would expect with the `?` end.

maybe create a new method (`#observers_list` or something?) that returns the list?